frontend/wallet/src/render/views/onboarding/RunNodeStep.re

```diff
@@ -29,7 +30,11 @@ module Styles = {
 };
 
 [@react.component]
-let make = (~prevStep, ~createAccount) => {
+let make = (~prevStep as _, ~createAccount, ~managed) => {
+  // Start the daemon process if we're in managed setup
+  let dispatch = React.useContext(ProcessDispatchProvider.context);
+  managed ? CodaProcess.useStartEffect(dispatch) : ();
```

Comment:
Calling a hook in a conditional https://reactjs.org/docs/hooks-rules.html

Reply:
Fixed.